internal/generator/templates/enum.go.tmpl

```diff
@@ -69,7 +69,7 @@ func (v {{ $enum.Name }}) String() string {
     if name, found := _{{ $enum.Name }}Name[v]; found {
         return name
     }
-    return fmt.Sprintf("unknown<%v>", v)
+    return fmt.Sprintf("unknown<%v>", {{ goType $scope $enum.Type }}(v))
```

Comment:
It would be great to have a test for that (i.e. the fact that `String` works).

Reply:
I've added a test!